src/include/denc.h

```diff
@@ -514,7 +514,7 @@ inline void denc_signed_varint_lowz(int64_t v, It& p) {
     v = -v;
     negative = true;
   }
-  unsigned lowznib = v ? (ctz(v) / 4) : 0u;
+  unsigned lowznib = v ? (std::countr_zero(std::bit_cast<uint64_t>(v)) / 4) : 0u;
```

Comment:
hi @tchaikov, can you please take a look at https://tracker.ceph.com/issues/57355?
Looks like this change is failing the windows tests.

Reply:
sure, i am staring at it.